pkg/recordset/recordset.go

```diff
@@ -225,6 +225,25 @@ func (m *Manager) createMissingTargetStacks(sourceStacks, targetStacks []string)
 }
 
 func (m *Manager) deleteOrphanTargetStacks(sourceStacks, targetStacks []string) error {
+	for _, target := range targetStacks {
```

Comment:
All this looks pretty much like there should be tests for it.

Reply:
Sure done, PTAL.